topics/rediscli.md

```diff
@@ -547,6 +547,10 @@ kind of objects, by key name:
     $ redis-cli --scan --pattern 'user:*' | wc -l
     3829433
 
+You can use `-i 0.01` to add a delay between calls to the `SCAN` command
+this will make the command slower but will significantly reduce load on
+the server.
```

Comment:
1. This is actually two sentenses ("... `SCAN` command. This will ...").
2. The code style for redis-doc is one sentence per line to avoid re-flowing text.

```suggestion
You can use `-i 0.01` to add a delay between calls to the `SCAN` command.
This will make the command slower but will significantly reduce load on the server.
```